src/apps/AppCenter/AppCenter.js

```diff
@@ -105,36 +105,53 @@ class AppCenter extends React.Component {
     const repos = this.getRepos()
     const currentRepo = openedRepoName && this.getRepoFromName(openedRepoName)
 
+    const navigationItems = [
+      'App Center',
+      ...(currentRepo ? [currentRepo.name] : []),
+    ]
+
     return (
       <React.Fragment>
         <AppView
+          style={{ height: '100%' }}
           appBar={
-            <AppBar
-              tabs={
-                openedRepoName ? null : (
-                  <TabBar
-                    items={SCREENS.map(screen => screen.label)}
-                    selected={activeTab}
-                    onChange={this.handleScreenChange}
+            <Viewport>
+              {({ below }) => (
+                <AppBar
+                  tabs={
+                    openedRepoName ? null : (
+                      <div
+                        css={`
+                          margin-left: ${below('medium') ? '-14px' : '0'};
```

Comment:
Ouch :(

Reply:
Yeah… I need to migrate `AppLayout` to aragonUI, so we can have a solid version everywhere. (I can’t use `AppLayout` with `AppCenter` yet because it’s using tabs… which why this hack was added).